triggers/daml/Daml/Trigger.daml

```diff
@@ -110,6 +112,22 @@ dedupCreate t = do
   unless (any ((Some t ==) . fromCreate) cmds) $
     void $ emitCommands [createCmd t] []
 
+-- | Create the template and exercise a choice on it it’s not already in the list of commands
+-- in flight (it will still be created if it is in the ACS).
+--
+-- Note that this will send the create as a single-command transaction.
+-- If you need to send multiple commands in one transaction, use
+-- `emitCommands` with `createCmd` and handle filtering yourself.
+dedupCreateAndExercise : (Eq t, Eq c, Template t, Choice t c r) => t -> c -> TriggerA ()
```

Comment:
fwiw I’m considering removing the `dedupX` functions completely in favor of the pending commands in `emitCommands`. While they seem convenient on simple examples, I think they also make it easy to get the condition you want to deduplicate on wrong and they don’t scale to multi-command transactions.

Let’s add it for now so we are consistent. If you have an opinion on this, I’d be interested in hearing it (probably not on this PR though, it’s completely unrelated :))